docs/sdk/web3-react.md

```diff
@@ -51,6 +51,15 @@ parameter is optional. If specified, the wallet provider will connect
 to an address in the specified group, otherwise the wallet provider
 will connect to address from any group.
 
+Note: When interacting with a deployed smart contract, use the same `addressGroup` that was specified during deployment. To identify the `addressGroup` of a deployed contract, use the following code:
+
+```typescript
+const nftCollection = NFTCollection.at(contractAddress);
+const index = await nftCollection.groupIndex;
+```
+
+This code retrieves the group index associated with the contract, confirming the correct `addressGroup`.
```

Comment:
Thanks for the update!

You can use the function `groupOfAddress(contractAddress)` from the SDK, which might be easier.

Reply:
Yes `groupOfAddress(contractAddress)` is an easier way to get the group of contract.